docs/usage/utils.md

```diff
@@ -0,0 +1,12 @@
+# utils
+
+## Get the HOST application URL using `getHostAppUrl`
+
+If you need to redirect a request to your app inside the user's admin you can use `getHostAppUrl`.
+
+```ts
+const redirectURL = getHostAppUrl(request);
+res.redirect(redirectURL);
+```
+
+This utility relies on the host query param being a Base 64 encoded string. All requests from Shopify should include this param in the correct format.
```

Comment:
@paulomarg  I wasn't really sure what to say here since the utility is so small.

Reply:
Yes, we can say that using `host` will ensure the redirect url is properly constructed and brings the merchant to the right place and is more reliably than using `shop`

Reply:
Updated 👍